### PR TITLE
Introduced new footprint type: `breakoutheaders`

### LINKED ISSUE #97: New footprint type: "breakoutheaders"
These are two columns of headers

### PR DESCRIPTION
**Pull request by AnasSarkiz**
/claim #97
/closes #97

### CODE REVIEW

**Review by Abse2001 — APPROVED** [review]
I think this looks good for me.

File: src/fn/breakoutheaders.ts [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+  if (params.right) {
+    return params.right * params.p
+  }
+
+  return 51
+}
+type Point = { x: number; y: number }
+type Direction = "left" | "right" | "top" | "bottom"
+
+const getTriangleDir = (
```
Comment (seveibar) [inline src/fn/breakoutheaders.ts:48]: this doesn't return a direction, but it sounds like it returns a triangle direction.

`getTrianglePath` with named parameters would be better!

**Review by seveibar — COMMENTED** [review]
minor naming issues, looks great tho!

File: src/fn/breakoutheaders.ts [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+  const height = params.h ?? getHeight(params)
+  const holes: PcbPlatedHole[] = []
+  const innerDiameter = params.id
+  const outerDiameter = params.od
+  let routes: { x: number; y: number }[] = []
+  if (params.right) {
+    const yoff = -((params.right - 1) / 2) * params.p
+    for (let i = 0; i < params.right; i++) {
+      if (i === 0 && !params.left && !params.bottom) {
+        routes = getTrianglePath(
```
Comment (seveibar) [inline src/fn/breakoutheaders.ts:101]:
```suggestion
        silkscreenTriangleRoutes = getTrianglePath(
```

**Review by seveibar — APPROVED** [review]

**Pull request merged**